bin/run.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
+
+set -e
+
+unset CDPATH
+cd "$( dirname "${BASH_SOURCE[0]}" )/.."
+
+echo() { builtin echo "[SCITRAN] $@"; }
+
+
+set -o allexport
+
+if [ "$#" -eq 1 ]; then
+    EXISTING_ENV=$(env | grep "SCITRAN_" | cat)
+    source "$1"
+    eval "$EXISTING_ENV"
+fi
```

Comment:
Well, cat's not necessary in that pipe, you can remove it.

In that order you describe, does last win or first win?